README.md

```diff
@@ -4,7 +4,7 @@
   <img src="2.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>FCC Test Suite Template</b></h3>
+  <h3><b>FreeCodeCamp Portfolio Project</b></h3>
```

Comment:
This is a much better title for the project as it is more descriptive of the contents of the project.